Ginger/Ginger/ValueExpression/ValueExpressionEditorPage.xaml.cs

```diff
@@ -181,13 +181,21 @@ private void BuildItemsTree()
                 }                
             }
 
-            TreeViewItem dataRoot = AddOrGetCategory("Data");
-            dataRoot.IsExpanded = true;
+            if (mObj != null && mObj.GetType() == typeof(FlowControl))
```

Comment:
Codacy found an issue: [This condition was just checked on line 171.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6400486)